app/bingey-ui/src/components/Watchlist/Watchlist.jsx

```diff
@@ -61,16 +61,16 @@ export const Watchlist = ({ titles }) => {
           Add title
         </Button>
       </div>
-      <Grid container spacing={2}>
-        {titles.map((title) => {
+      <Grid container spacing={3}>
+        {titles.map((titles) => {
```

Comment:
This might cause a lot of reading confusion even if it compiles correctly. Why not
```suggestion
        {titles.map((title) => {
``` 
?